CDP4Common/Poco/Option.cs

```diff
@@ -40,5 +47,22 @@ public bool IsDefault
                 return iteration?.DefaultOption == this;
             }
         }
+
+        /// <summary>
+        /// Finds <see cref="NestedParameter"/>s by their <see cref="NestedParameter.Path"/>s in the <see cref="Option"/>'s <see cref="NestedParameter"/>
+        /// and returns its <see cref="NestedParameter.ActualValue"/> "converted" to the generic <typeparamref name="T"></typeparamref>'s .
```

Comment:
nit: I think <typeparamref>could be shortened to a self-closing tag.
nit: there's an extra whitespace before the final dot.